.github/workflows/push-browsermt-main-to-mozilla-main.yml

```diff
@@ -0,0 +1,27 @@
+name: Push browsermt/main branch to mozilla/bergamot-translator
+
+on:
+  schedule:
+  # Run every hour
+  - cron: "0 * * * *"
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Mirror a branch from a remote repo into this one
+    env:
+      source_repository: 'browsermt/bergamot-translator'
+      this_repository: 'mozilla/bergamot-translator'
+      branch: 'main'
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Mirror a branch from a remote repo into this one
+        if: github.repository == '${{ env.this_repository }}'
```

Comment:
I'm hoping this works. I just got bit by an issue in my source repository with failures because

> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression, unless the expression contains any operators. If the expression contains any operators, the expression must be contained within ${{ }} to explicitly mark it for evaluation. For more information, see "Context and expression syntax for GitHub Actions."

I'd strongly recommend testing `test-branch` -> `test-branch` updating it with master and ensuring it works before a merge to avoid debugging later.

Reply:
> I'd strongly recommend testing test-branch -> test-branch updating it later with main -> main and ensuring it works before a merge to avoid debugging later.

I'd like to do that, but can it be tested without merging? Or are you suggesting that I also set up two separate repos and apply it there first?

Reply:
I think what is being suggested is to have `source/temp_branch` pushed to `target/temp_branch` such that the main branch isn't altered in the event it doesn't work as expected.

However, in order to use `schedule`, the workflow must be on the default branch of the repository. I believe the same* is true of `workflow_dispatch`. In either case, it would need this file merged to the default branch.

*the workflow has to exist on the default branch, but can be run against other branches